src/app/(auth)/layout.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+
+export default function AuthLayout({
+    children,
+}: {
+    children: React.ReactNode;
+}) {
+    return (
+        <main className="flex  flex-col items-center   grow">
+            <div className=" flex   ">
```

Comment:
There are some extra spaces :)
```suggestion
        <main className="flex flex-col items-center grow">
            <div className="flex">
```

There are same `classNames` in other places